src/Butler/App/RandomCat.hs

```diff
@@ -0,0 +1,76 @@
+module Butler.App.RandomCat (randomCatApp) where
+
+import Butler
+import Network.HTTP.Client (
+    Manager,
+    Response (responseBody, responseStatus),
+    brConsume,
+    parseRequest_,
+    withResponse,
+ )
+import Network.HTTP.Client.TLS (newTlsManager)
+import Network.HTTP.Types (statusIsSuccessful)
+
+import Data.ByteString.Base64 qualified as B64 (encode)
+import Data.ByteString.Char8 qualified as C8
+
+randomCatApp :: App
+randomCatApp =
+    (defaultApp "RandomCat" startRandomCat)
+        { tags = fromList ["Utility"]
+        , description = "Display a random cat using CATAAS"
+        , size = Just (500, 500)
+        }
+
+data RandomCatState = Loading | LoadingError | Loaded ByteString
+
+startRandomCat :: AppContext -> ProcessIO ()
+startRandomCat ctx = do
+    -- Setup state
+    logInfo "RandomCat started!" []
+    state <- newTVarIO Nothing
```

Comment:
nit: using a `Unknown` cat state instead of wrapping with a Maybe also works. That makes the pattern match simpler.